ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/health/HealthService.scala

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.health
+
+import akka.Done
+import akka.stream.Materializer
+import akka.stream.scaladsl.{Sink, Source}
+import com.digitalasset.grpc.{GrpcException, GrpcStatus}
+import com.digitalasset.platform.sandbox.health.HealthService._
+import io.grpc.health.v1.{HealthCheckRequest, HealthCheckResponse, HealthGrpc}
+import io.grpc.stub.StreamObserver
+import io.grpc.{Context, Status}
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.{DurationInt, FiniteDuration}
+import scala.util.control.NonFatal
+import scala.util.{Failure, Success}
+
+class HealthService(watchThrottleFrequency: FiniteDuration = 1.second)(
+    implicit materializer: Materializer,
+    executionContext: ExecutionContext
+) extends HealthGrpc.HealthImplBase {
```

Comment:
I'd prefer if we could stick with the scalapb rpc implementation for services, otherwise we start having a mix of how services are implemented.

Reply:
On it.

Reply:
Done.